pages.nl/linux/archinstall.md

```diff
@@ -0,0 +1,12 @@
+# archinstall
+
+> Guided Arch Linux installer with a twist.
```

Comment:
Are you sure this is well translated? :eyes:

Reply:
Forgot the whole header. Whoopsie.